app/controllers/sessions_controller.rb

```diff
@@ -42,7 +42,7 @@ def signout
   end
 
   def failure
-    redirect_to "/401"
+    redirect_to unauthorized_path
```

Comment:
Should this be redirecting or just responding with a 401?

Reply:
This is the frontend so yes, we should redirect.

Reply:
We respond with a 404 and show page not found for that URL, we don't redirect to `/404`.
Why isn't this the same?